### PR TITLE
Fix #25472: Fix Path for Material Icons in Edit Mode

### LINKED ISSUE #25472: Editing a Page is Loading the Material Icons Relatively
### Problem Statement
Currently, when users edit a page in dotCMS, the material icons load relatively instead of absolutely, which causes the page to not render correctly.

### Steps to Reproduce
1. Login to dotCMS.
2. Edit an existing page.
3. Observe that the material icons are loading relatively instead of absolutely.

### Acceptance Criteria
- All references to material icons should be loading absolutely.
- All pages should render correctly with the material icons loading absolutely.

### External Links
N/A

### Assumptions & Initiation Needs
- We assume that the reference to material icons is in the front end code.

### Quality Assurance Notes & Workarounds
- To test that the material icons are loading correctly, users can preview the page and observe the rendering.

### PR DESCRIPTION
**Pull request by zJaaal**
### Summary
🐛🎨🔗

Fixed a bug in the dotCMS edit page UI where the material icons were not loading correctly. Updated the `createLinkElement` method in `dot-edit-content-html.service.ts` to use an absolute path for the material-icons.css file.

> _`createLinkElement`_
> _Fixes icon bug with absolute path_
> _A winter solstice_

### Walkthrough
*  Fixed a bug where the material-icons.css file was not loading correctly in some scenarios by using an absolute path instead of a relative path in the `createLinkElement` method ([link](https://github.com/dotCMS/core/pull/25491/files?diff=unified&w=0#diff-803b27038de2abbfb0091318308d39df3f46e46241dc50eeb2271abeb1b45ae7L424-R424))

#### Original

https://github.com/dotCMS/core/assets/63567962/aed77a87-a71e-4db0-a9f8-6f3f1eb4bcc3

#### Updated

https://github.com/dotCMS/core/assets/63567962/feee65e8-1806-4c6d-a72a-8ddf1bd9adcb

### CODE REVIEW

**Review by manuelrojas — APPROVED** [review]

**Review by oidacra — APPROVED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts [hunk 1]
```diff
@@ -421,7 +421,7 @@ export class DotEditContentHtmlService {
     private setMaterialIcons(): void {
         const doc = this.getEditPageDocument();
         const link = this.dotDOMHtmlUtilService.createLinkElement(
-            'dotAdmin/assets/material-icons.css'
+            '/dotAdmin/assets/material-icons.css'
         );
```
Comment (rjvelazco) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts:425]: Create a variable for this, and can we test this `path`? That way, we avoid having the same `issue` if someone accidentally changes it.

**Review by rjvelazco — APPROVED** [review]

**Review by rjvelazco — APPROVED** [review]

**Review by hmoreras — APPROVED** [review]

**Review by rjvelazco — CHANGES_REQUESTED** [review]
See my comment

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts (hunk 1 above)
Comment (zJaaal) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts:425]: This service has all the test skipped, so I'm not really sure about the testing, but I can add the constant

**Review by rjvelazco — APPROVED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts [hunk 2]
```diff
@@ -420,9 +422,7 @@ export class DotEditContentHtmlService {
 
     private setMaterialIcons(): void {
         const doc = this.getEditPageDocument();
-        const link = this.dotDOMHtmlUtilService.createLinkElement(
-            'dotAdmin/assets/material-icons.css'
-        );
+        const link = this.dotDOMHtmlUtilService.createLinkElement(MATERIAL_ICONS_PATH);
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts:425]: Can we add a tests here?

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts (hunk 2 above)
Comment (zJaaal) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts:425]: Tests are too unstable to be unskipped.

**Pull request merged**